script.js

```diff
@@ -1,14 +1,150 @@
-const toolbarBtn = document.getElementById('toolbar-btn');
 const menu = document.getElementById('menuOpen');
 const mainContainer = document.getElementById('mainContainer');
-const cancelBtn = document.getElementById('closeBtn');
 
 function menuOpen() {
-    menu.style.display = "flex";
-    mainContainer.style.display = "none";
+  menu.style.display = "flex";
+  mainContainer.style.display = "none";
 }
 
 function menuClose() {
-    menu.style.display = "none";
-    mainContainer.style.display = "block";
-}
\ No newline at end of file
+  menu.style.display = "none";
+  mainContainer.style.display = "block";
+}
+
+const element = document.getElementById("popUpCard");
+const actionBtn = document.querySelectorAll('.action');
+const workCards = document.querySelectorAll('.frame');
+const headerTwo = document.createElement("h2");
+const popupList = document.createElement("ul");
+const image = document.createElement("img");
+const content = document.createElement("p");
+const container = document.createElement("div");
+const division = document.createElement("div");
+const closeBtn = document.createElement("button");
+const btnImages = [{
+  name: "See Live",
+  path: "Assets/Icon-live.png",
+},
+{
+  name: "See Source",
+  path: "Assets/git-white.png",
+}];
+
+headerTwo.classList.add('popup-header');
+popupList.classList.add('popup-list');
+image.classList.add('popup-image');
+content.classList.add('popup-text');
+division.classList.add('popup-btn-container');
+closeBtn.classList.add('popup-close-btn');
+
```

Comment:
- [ ] I don't see any project data here in your javascript file. From the project requirements, you are expected to store project data as an array of objects so that you will be able to loop through it while rendering them dynamically 😎 
Please add the project's data here so that you will be able to iterate through them 🏃 👍 

![image](https://user-images.githubusercontent.com/90222110/189175503-6da90034-a6b9-4960-b87d-017427f48f69.png)

Reply:
I stored them in a separate JavaScript sheet named (data.js).